### PR TITLE
fix `FileListWrapper breaks some client code` (close #689)

### LINKED ISSUE #689: FileListWrapper breaks some client code
The following code is used to check browser support of the `input.files` property:
```javascript
var input = document.createElement('input');
input.type = 'file';
var supportInputFiles = input.files instanceof FileList
```

After proxying this check fails because `input.files` has FileListWrapper constructor.

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

File: test/client/fixtures/sandbox/upload-test.js [hunk 1]
```diff
@@ -764,3 +764,13 @@ if (browserUtils.isIE) {
...
     });
 }
+
+if (window.FileList) {
+    test('the "instanceof FileList" operation works correctly with FileListWrapper instances (GH-689)', function () {
+        var input = document.createElement('input');
+
+        input.type = 'file';
+
+        ok(eval(processScript('input.files instanceof FileList')));
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/upload-test.js]: You can write quite more simple `ok(getProperty(input.files instanceof FileList)`.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
lgtm

**Pull request merged**